private/buf/bufctl/controller.go

```diff
@@ -378,8 +378,8 @@ func (c *controller) GetTargetImageWithConfigs(
 		if err != nil {
 			return nil, err
 		}
-		lintConfig := bufconfig.DefaultLintConfigV1
-		breakingConfig := bufconfig.DefaultBreakingConfigV1
+		lintConfig := bufconfig.DefaultLintConfigV2
```

Comment:
We might want to leave this as V1 here for backwards compatibility (if no config file is found).

I also noticed that it appears that `DefaultLintConfigV2` isn't passing the right value for `allowCommentIgnores` - it should be `true` I think to match the new behavior when parsing from the external config file?

Reply:
Yeah, it's fair for back compat to set this back to `v1`. And yes, made an adjustment to `v2` defaults :)